composer.lock

```diff
@@ -5837,5 +5834,5 @@
     "platform-overrides": {
         "php": "7.3"
     },
-    "plugin-api-version": "2.1.0"
```

Comment:
I'm not sure about this plugin API version number being bumped. I think we should stay on composer v2.1